fleetspeak/src/server/mysql/clientstore.go

```diff
@@ -356,7 +356,11 @@ func (d *Datastore) RecordResourceUsageData(ctx context.Context, id common.Clien
 			rud.ResourceUsage.MeanSystemCpuRate,
 			rud.ResourceUsage.MaxSystemCpuRate,
 			int32(rud.ResourceUsage.MeanResidentMemory*bytesToMIB),
-			int32(float64(rud.ResourceUsage.MaxResidentMemory)*bytesToMIB))
+			int32(float64(rud.ResourceUsage.MaxResidentMemory)*bytesToMIB),
+			int32(rud.ResourceUsage.MeanIoRead*bytesToMIB),
+			int32(float64(rud.ResourceUsage.MaxIoRead)*bytesToMIB),
```

Comment:
Would it make sense to round the value to the closest integer before converting to int32?

Reply:
Following an offline discussion with @mbushkov, we'll round up the the first integer _x_ such that `MeanIo(Read|Write) <= x`, and convert the value to `int64` instead of `int32`.